src/addons/devkit/addon.cpp

```diff
@@ -1593,6 +1655,14 @@ void RenderCapturePane(reshade::api::device* device, DeviceData* data) {
 
       if (ImGui::TableSetColumnIndex(CAPTURE_PANE_COLUMN_REF)) {
         ImGui::Text("%03d", draw_index);
+
+        // If a pending focus request targets this draw index, set focus/scroll here
+        const int pending = snapshot_focus_pending_index;
+        if (pending >= 0 && pending == draw_index) {
```

Comment:
No reason to create a `pending` variable and check range. `draw_index` can't be negative and and no selected index means `snapshot_focus_pending_index` isn't `-1` and thus will never match. It's enough with

`if (snapshot_focus_pending_index == draw_index) {`